### ISSUE TITLE
Wildly different quantization performance on different DenseNet169 models

### ISSUE THREAD

**Issue by wickstopher** [tensorflow/tensorflow#23352]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): macOS 10.14 Mojave
- TensorFlow installed from (source or binary): binary (pip)
- TensorFlow version (use command below): 1.11.0
- Python version: 3.6.4

**Describe the current behavior**
I have two models that I have trained using Keras. The two models use the same architecture (Keras' DenseNet169 implementation), however have a different number of target classes (80 in one case, 200 in the other case).

* Converting both models to .pb format works just fine (identical performance in inference).

* Converting both models to .tflite format using TOCO works just fine (again, identical performance in inference).

* Converting the 80-class model to .tflite using quantization in TOCO works reasonably well (~1% drop in top 3 accuracy).

* Converting the 200-class model to .tflite using quantization in TOCO goes off the rails (~30% drop in top 3 accuracy).

I'm using an identical conversion process for both models.

`toco --graph_def_file frozen_graph.pb --output_file quantized_graph.tflite --inference_type FLOAT --inference_input_type FLOAT --output_format TFLITE --input_arrays input_1 --output_arrays output_node0 --quantize True`

**Describe the expected behavior**

I would expect that two models trained to similar accuracy on the same architecture should both quantize reasonably well, despite the number of prediction target classes in the fully-connected layers.

**Comment by wickstopher** [comment]
closing this issue, as I don't think it really belongs here. Have opened a StackOverflow question instead: https://stackoverflow.com/questions/53050923/wildly-different-quantization-performance-on-tensorflow-lite-conversion-of-keras/53056589#53056589
